tasks/databases.rake

```diff
@@ -6,7 +6,12 @@ namespace :db do
   namespace :migrate do
     desc "Migrate the database data and schema (options: VERSION=x, VERBOSE=false)."
     task :with_data => :environment do
-      original_db_config = ActiveRecord::Base.connection_db_config
+      original_db_config = if Gem::Dependency.new("rails", "~> 7.0").match?("rails", Gem.loaded_specs["rails"].version)
+        ActiveRecord::Base.connection_db_config
+      elsif Gem::Dependency.new("rails", "~> 6.0").match?("rails", Gem.loaded_specs["rails"].version)
+        ActiveRecord::Base.connection_config
+      end
```

Comment:
@ilyakatz This one has just introduced the same issue that was fixed in #260 :)

Reply:
oh, blah, sorry, missed that. created a PR for this https://github.com/ilyakatz/data-migrate/pull/263 

FYI @chaunce

Reply:
does that look right to you?